cmd/cli/mesh_upgrade_test.go

```diff
@@ -5,7 +5,6 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"helm.sh/helm/v3/pkg/action"
 	helm "helm.sh/helm/v3/pkg/action"
```

Comment:
Generally I feel its not worth aliasing these imports if their package names are not versioned (`action` here vs `v1`). Could you remove the non aliased import instead and use `action.` in the code?